go/client/cmd_log_send.go

```diff
@@ -122,6 +126,9 @@ func (c *CmdLogSend) post(status, kbfsLog, svcLog, desktopLog string) error {
 	if err := addFile(mpart, "gui_log_gz", "gui_log.gz", desktopLog); err != nil {
 		return err
 	}
+	if err := addFile(mpart, "start_log_gz", "start_log.gz", startLog); err != nil {
```

Comment:
Not yet... is this all thats required?
https://github.com/keybase/keybase/pull/593